bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/SecurityCapability.java

```diff
@@ -110,7 +116,55 @@ protected void updateHomeEvent(HomeEvent homeEvent) {
                 });
     }
 
-    public Collection<HomeEvent> getDeviceEvents(String cameraId, String deviceType) {
+    @Override
+    protected List<NAObject> updateReadings(SecurityApi api) {
```

Comment:
Just one question: are we sure that this method will be always called before any call to `updateReadings` in CameraCapability and PersonCapability ?

Reply:
Should be but not guaranteed 100% but this should not be an issue because when requested event is not in the buffer it is grabbed especially for the device. Do you see a potential issue here ?

Reply:
The issue I see is that the event for person / camera could then be not the most recent, rather the last present in the cache.

Reply:
Yes, it will work in case the home events are always polled before the camera and the person are updated.
In case the camera/person refresh is (more or less) independent from the home, you could get provide an outdated event.
That was my origin question.

Reply:
Ok, I get your question, but I don't know how to address it. You're correct, until next home refresh, we're suspect to provide outdated events, but then I don't see how to avoid emitting a new request. It should not be the case when using webhooks, but apart from that...your ideas are welcome.

Reply:
I do not remember exactly how you manage that. Don't you trigger the refresh of the home, the camera devices and the persons at the same frequency, more or less at the same time. Maybe even in parallel ?
If that is correct, one solution, if doable, would be to force the home refresh being finished before refreshing the camera devices and persons.

Reply:
According to CommonInterface.java : 

```
    default List<NAObject> updateReadings() {
        List<NAObject> result = new ArrayList<>();
        getCapabilities().values().forEach(cap -> result.addAll(cap.updateReadings()));
        getActiveChildren().forEach(child -> result.addAll(child.updateReadings()));
        return result;
    }
```
Home will be updated (Api requests sent) in sequence just before childs (modules, cameras, persons). So I do not think this should generate outdated data in some milliseconds between the two events.

Reply:
Good